backend/config/config.go

```diff
@@ -438,3 +438,20 @@ func migrateUserLanguage(appconf *AppConfig) {
 		delete(frontconf, "userLanguage")
 	}
 }
+
+// migrateActiveTokens removes tokens from AccountsConfig
+func migrateActiveTokens(accountsConf *AccountsConfig) {
+	for _, account := range accountsConf.Accounts {
+		// copy account so it does not take a reference to the loop scoped account
+		ac := account
+		if ac.CoinCode != coin.CodeETH {
+			continue
+		}
+
+		err := ac.SetTokenActive("eth-erc20-sai0x89d", false)
```

Comment:
I never really worked with migrations (when I did they were created automatically) it feels wrong to hard code it like this but I guess this is what we need to do here ? @benma and in case there are other tokens added and then removed they would also need to be removed here ?

Reply:
Well it's not a database with a schema or its own query language like SQL, it is just a json file. Maybe we can check if there are some tools overlaying a schema and automatic migrations, but that would be very low prio - what we have is working well enough.

I think this is the first token to be removed.